instrumentation/vertx-core-4.3.2/build.gradle

```diff
@@ -0,0 +1,25 @@
+jar {
+    manifest {
+        attributes 'Implementation-Title': 'com.newrelic.instrumentation.vertx-core-4.3.2'
+    }
+}
+
+
+
+dependencies {
+    implementation(project(":agent-bridge"))
+    implementation("io.vertx:vertx-core:4.3.2")
+    testImplementation("io.vertx:vertx-core:4.3.2")
+}
+
+verifyInstrumentation {
+    passesOnly 'io.vertx:vertx-core:[4.3.2,)'
+    excludeRegex '.*CR[0-9]*'
```

Comment:
Can you remove these excludes?
Or at least let the 5.0.0* be run.
This way we will be warned that the extension will not work with the new version a little before it is released.

For the older modules it is not a big deal, since we don't expect them to apply.